Description: Remove google custom engine queries script
 Remove javascript for the custom google engine queries, in the header
 and in the footer of each html documentation page, to avoid potential
 privacy breach.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Origin: other
Forwarded: not-needed
Last-Update: 2016-09-29

--- a/doc/html/header.txt
+++ b/doc/html/header.txt
@@ -5,7 +5,6 @@
 	<meta name="viewport" content="width=device-width" />
 
 	<!--#pagetitle-->
-	<!--#canonical-->
 
 	<link rel="shortcut icon" href="favicon.ico" />
 
@@ -15,26 +14,6 @@
 	<link rel="stylesheet" type="text/css" href="slurm.css" />
 
 	<script src="jquery.min.js"></script>
-	<script type="text/javascript">
-	jQuery(document).ready(function() {
-		jQuery('.menu-trigger').bind('click touchstart', function() {
-			jQuery(this).find('.menu-trigger__lines').toggleClass('menu-trigger__lines--closed');
-			jQuery(this).parents('.site-header').find('.site-nav').toggleClass('site-nav--active');
-
-			return false;
-		});
-	});
-
-	(function() {
-	  var cx = '011890816164765777536:jvrtxrd3f0w';
-	  var gcse = document.createElement('script');
-	  gcse.type = 'text/javascript';
-	  gcse.async = true;
-	  gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
-	  var s = document.getElementsByTagName('script')[0];
-	  s.parentNode.insertBefore(gcse, s);
-	})();
-	</script>
 </head>
 
 <body>
--- a/doc/html/footer.txt
+++ b/doc/html/footer.txt
@@ -11,18 +11,5 @@
 	</nav>
 </footer>
 
-<script type='text/javascript'>
-	var custpagename = window.location.href;
-	var urlarray = custpagename.split('#');
-	custpagename = urlarray[1];
-
-	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-			m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-				})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-			 ga('create', 'UA-47927131-1', 'schedmd.com');
-		ga('send', {'hitType': 'pageview', 'page': custpagename, 'title': custpagename});
-</script>
-
 </body>
 </html>
